modules/project-docs/pages/sdk-release-notes.adoc

```diff
@@ -13,15 +13,26 @@ These pages cover the 4.x versions of the Couchbase PHP SDK.
 For download instructions, see the xref:sdk-full-installation.adoc[installation page].
 
 
-== Version 4.0.0 (XXX April 2022)
+== Version 4.0.0 (11 May 2022)
 
-Version 4.0.0 is the first major release of the next generation PHP SDK, built on the Couchbase++ library -- featuring multi-document distributed ACID transactions, and bringing a number of improvements related to internal connection behavior.
+Version 4.0.0 is the first major release of the next generation PHP SDK, built on the Couchbase++ library -- featuring
+multi-document distributed ACID transactions, and bringing a number of improvements related to internal connection
+behavior.
 
-API documentation: https://docs.couchbase.com/sdk-api/couchbase-php-client-4.0.0/
+API documentation: https://docs.couchbase.com/sdk-api/couchbase-php-client-4.0.0
 
-* https://issues.couchbase.com/browse/PCBC-123[PCBC-123]: 
-Sample ticket XXXXXX
+|===
+|Platform|
```

Comment:
@avsej what Platform did we actually intend to put here?

Reply:
in previous version of the release notes, this column also contained combinations of Windows platforms (like PHP version, architecture and thread safety variant). For this, 4.0.0 release we don't publish binaries for windows, so this column does not represent any "platform". Feel free to remove this table header completely.